service/src/test/java/uk/gov/hmcts/reform/fpl/service/DocumentDownloadServiceTest.java

```diff
@@ -83,7 +88,7 @@ public void shouldDownloadDocumentFromDocumentManagement() throws Exception {
             eq(join(",", CAFCASS.getRoles())), anyString(), anyString()))
             .willReturn(resourceResponseEntity);
 
-        byte[] documentContents = documentDownloadService.downloadDocument(token, userId, document.links.binary.href);
+        byte[] documentContents = documentDownloadService.downloadDocument(document.links.binary.href);
```

Comment:
now you have to change verification at the end of test
verify(documentDownloadClient).downloadBinary(anyString(), anyString(),
to
verify(documentDownloadClient).downloadBinary(authToken, userId

as now correct usage of RequestData is not verified